src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -78,13 +87,26 @@ public PublicationController(PublicationService publicationService) {
         @RequestHeader(value = PublicationConfiguration.DISPLAY_TO_HEADER, required = false)
         @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime displayTo,
         @RequestBody String payload) {
-        validateRequestHeaders(provenance, sourceArtefactId);
+
+        Map<String, Object> headers = new HashMap<>();
```

Comment:
Rather than putting it in a map, I'm wondering if it's better to create a model that represents the headers. This means that we keep the type safety, and prevents us from casting in the validation utils class

Reply:
Done, this does seem to be a nicer approach - cuts down on code and massively reduces that in test